i18n/english.yml

```diff
@@ -90,6 +90,12 @@ components:
     missingNameAlert: Must give snapshot a valid name!
     ok: OK
     title: Create a new snapshot
+  CustomCSVForm:
+    addCsvData: "Add the CSV data to add to/replace in the incoming GTFS:"
+    addCustomCsvData: Add the custom CSV data.
+    numLines: "%numLines% lines."
+    saveCsv: Save CSV
+    saveCsvAndFileName: Save CSV and file name
```

Comment:
Why are these separate?

Reply:
Within a ReplaceFileFromString, there is no file name to save. For adding a custom csv, it's saving both the csv data and the file name. Should I reorganize this somehow?